src/lib/application/application.schema.d.ts

```diff
@@ -40,4 +40,8 @@ export interface ApplicationOptions {
    * Nest included development dependencies (comma separated values).
    */
   devDependencies?: string;
+  /**
+   * Case format. Options are 'kebab' | 'snake' | 'camel' | 'pascal'.
```

Comment:
Are you missing `kebab-or-snake` option?

Reply:
No, I would prefer to deprecate it ASAP. I wouldn't expose it to users.